topostats/run_topostats.py

```diff
@@ -264,7 +264,7 @@ def process_scan(
                 plotting_config["plot_dict"][plot_name]["output_dir"] = Path(_output_dir)
                 Images(
                     filtered_image.images["gaussian_filtered"],
-                    filename=filtered_image.filename + "_processed_masked",
+                    filename=filtered_image.filename + f"_{direction}_processed_masked",
```

Comment:
Could just be a single f-string

```suggestion
                    filename=f"{filtered_image.filename}_{direction}_processed_masked",
```